sklearn_pandas/features_generator.py

```diff
@@ -25,6 +25,9 @@ def gen_features(columns, classes=None):
 
                 If None value selected, then each feature left as is.
 
+    arguments   a dictionary of additional values such as {'prefix': 'x',
```

Comment:
If possible, better to specify the explicit arguments accepted in the signature of the function. I believe this is OK even if there are a lot of arguments - sklearn has this and it is OK.

They are `alias`, `prefix`, and `suffix`, right?

Reply:
yes...will fix this